components/svg/plugin_icon.jsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Do you mind to move it to `components/widgets/icons/`?

Reply:
Done